crates/primitives/block/src/lib.rs

```diff
@@ -151,7 +156,7 @@ impl DeoxysBlockInfo {
     }
 }
 
-#[derive(Clone, Debug, serde::Serialize, serde::Deserialize)]
+#[derive(Clone, Debug, Default, PartialEq, Eq, serde::Serialize, serde::Deserialize)]
```

Comment:
Could you add some rustdoc to your new structures.
In the future we would like to have a proper rustdoc deployed.

Reply:
Done